### PR TITLE
Chore/fix redis rdb policy logic

### LINKED ISSUE #37522: [bitnami/redis-cluster] The save config is overridden due to wrong logic in libredis.sh
### Name and Version

redis-cluster

### What architecture are you using?

None

### What steps will reproduce the bug?

1. Add save config in the default redis config file, for example, `save 3600 1 900 100 180 10000`
2. Leave env `REDIS_RDB_POLICY` empty
3. Start the container with mounting the config file

### What is the expected behavior?

the `save` config in config file in container should be  `save 3600 1 900 100 180 10000`

### What do you see instead?

the last `save` config in config file in container became
```
save 3600 1 900 100 180 10000
save ""
```
because environment variable `REDIS_RDB_POLICY` is empty and the logic in [this commit](https://github.com/bitnami/containers/pull/29381/files#diff-0986414641a98c533d39a0845bb142b4caba106615da84b851448e3113c74826R409-R410) would just append the `save ""` to the config file, hence override the save config we have in config file

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by dogzzdogzz**
### Applicable issues
- fixes #37522

### CODE REVIEW

File: bitnami/redis-cluster/6.2/debian-11/rootfs/opt/bitnami/scripts/libredis.sh [hunk 1]
```diff
@@ -406,12 +406,14 @@ redis_configure_default() {
...
-          redis_conf_set save ""
-        else
-          for i in ${REDIS_RDB_POLICY};do
-            redis_conf_set save "${i//#/ }"
-          done
+        if [ ! -z "${REDIS_RDB_POLICY}" ];then
+            if [ "${REDIS_RDB_POLICY}" == "disabled" ]; then
+                redis_conf_set save ""
+            else
+                for i in ${REDIS_RDB_POLICY};do
```
Comment (corico44) [inline bitnami/redis-cluster/6.2/debian-11/rootfs/opt/bitnami/scripts/libredis.sh:413]: Could you leave a space after ; please?

File: bitnami/redis-cluster/7.0/debian-11/rootfs/opt/bitnami/scripts/libredis.sh [hunk 2]
```diff
@@ -406,12 +406,14 @@ redis_configure_default() {
         redis_conf_set appendonly "${REDIS_AOF_ENABLED}"
 
         #The value stored in $i here is the number of seconds and times of save rules in redis rdb mode
-        if [ -z "${REDIS_RDB_POLICY}" ];then
-          redis_conf_set save ""
-        else
-          for i in ${REDIS_RDB_POLICY};do
-            redis_conf_set save "${i//#/ }"
-          done
+        if [ ! -z "${REDIS_RDB_POLICY}" ];then
```
Comment (corico44) [inline bitnami/redis-cluster/7.0/debian-11/rootfs/opt/bitnami/scripts/libredis.sh:409]: Could you leave a space after ; please?

File: bitnami/redis-cluster/7.0/debian-11/rootfs/opt/bitnami/scripts/libredis.sh [hunk 3]
```diff
@@ -406,12 +406,14 @@ redis_configure_default() {
...
+            if [ "${REDIS_RDB_POLICY}" == "disabled" ]; then
+                redis_conf_set save ""
+            else
+                for i in ${REDIS_RDB_POLICY};do
```
Comment (corico44) [inline bitnami/redis-cluster/7.0/debian-11/rootfs/opt/bitnami/scripts/libredis.sh:413]: Could you leave a space after ; please?

Comment (dogzzdogzz) [reply bitnami/redis-cluster/7.0/debian-11/rootfs/opt/bitnami/scripts/libredis.sh]: resolved

File: bitnami/redis-cluster/7.0/debian-11/rootfs/opt/bitnami/scripts/libredis.sh (hunk 2 above)
Comment (dogzzdogzz) [reply bitnami/redis-cluster/7.0/debian-11/rootfs/opt/bitnami/scripts/libredis.sh]: resolved

File: bitnami/redis-cluster/6.2/debian-11/rootfs/opt/bitnami/scripts/libredis.sh (hunk 1 above)
Comment (dogzzdogzz) [reply bitnami/redis-cluster/6.2/debian-11/rootfs/opt/bitnami/scripts/libredis.sh]: resolved

File: bitnami/redis-cluster/6.2/debian-11/rootfs/opt/bitnami/scripts/libredis.sh [hunk 4]
```diff
@@ -406,12 +406,14 @@ redis_configure_default() {
         redis_conf_set appendonly "${REDIS_AOF_ENABLED}"
 
         #The value stored in $i here is the number of seconds and times of save rules in redis rdb mode
-        if [ -z "${REDIS_RDB_POLICY}" ];then
-          redis_conf_set save ""
-        else
-          for i in ${REDIS_RDB_POLICY};do
-            redis_conf_set save "${i//#/ }"
-          done
+        if [ ! -z "${REDIS_RDB_POLICY}" ];then
```
[the remark this answers is not in the document]
Comment (dogzzdogzz) [reply bitnami/redis-cluster/6.2/debian-11/rootfs/opt/bitnami/scripts/libredis.sh]: resolved

**Comment by corico44** [comment]
Hello @dogzzdogzz. Thanks for creating this PR. Please, note it is needed to sign the commits. Your signature certifies that you wrote the patch or otherwise have the right to contribute the material. You can find more info about this topic, in [the “Sign your work” section from the contributing guidelines](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work).

**Comment by dogzzdogzz** [comment]
> 

@corico44 Thanks for the reminding, the commits have been signed off

**Review by corico44 — APPROVED** [review]
LGTM

**Pull request merged**